doc/development.md

```diff
@@ -115,6 +115,15 @@ For this checkout the project with the current feature branch:
 git clone https://github.com/vector-im/element-call.git
 git checkout feature-sfu
 ```
+
+In case you'll develop on the _Matrix JS SDK_ you'll also want to yarn link the js-sdk like
+```bash
+git clone https://github.com/matrix-org/matrix-js-sdk.git
```

Comment:
Also please checkout the correct branch - `matthew/sfu`, I think? Sorry for not being clear ealier

Reply:
👍